packages/ListBox/stories/ListBox.mdx

```diff
@@ -0,0 +1,284 @@
+
+```js
+  <ListBox height={320}>
+    {images.map(image => (
+      <ListBox.Option key={image.src} label={image.label}>
+        <ImageOption>
+          <img alt={image.label} src={image.src} />
+        </ImageOption>
+      </ListBox.Option>
+    ))}
+  </ListBox>
+```
+
+#### Result
+
+<ListBox height={320}>{images.map(image => (<ListBox.Option key={image.src} label={image.label}><ImageOption><img alt={image.label} src={image.src} /></ImageOption></ListBox.Option>))}</ListBox>
+
+### Adding inline to the ListBox
+
+You can add inline to your ListBox to prevent the listBox from collapsing the list options. Simply add the ```isInline``` prop to the ListBox. 
+
+```js
+  <ListBox isInline>
+    <ListBox.Popover zIndex={10000}>Anti-Heroes</ListBox.Popover>
+    {characters.antiHeroes}
+  </ListBox>
+```
+
+#### Result
+
+<ListBox isInline><ListBox.Popover zIndex={10000}>Anti-Heroes</ListBox.Popover>{characters.antiHeroes}</ListBox>
+
+### Adding disable to the ListBox
+
+You can disable ListBox options that you dont want users to be able to select. Simply add the ```isDisabled``` prop to the ```ListBox.option``` that you wish to disable. 
+
+```js
+  <ListBox>
+    <ListBox.Option isDisabled >Option 1</ListBox.Option>
+    <ListBox.Option>Option 2</ListBox.Option>
+  </ListBox>
+```
+
+#### Result
+
+<ListBox><ListBox.Option isDisabled >Option 1</ListBox.Option><ListBox.Option>Option 2</ListBox.Option></ListBox>
+
+### Adding an empty option to the ListBox
+
+You can add an empty option to the ListBox by setting the ```label``` property to an empty string.
+
+```js
+  <ListBox>
+    <ListBox.Option label="">&nbsp;</ListBox.Option>
+    {characters.heroes}
+    {characters.villians}
+    {characters.antiHeroes}
+  </ListBox>
+```
+
+#### Result
+
+<ListBox><ListBox.Option label="">&nbsp;</ListBox.Option>{characters.heroes}{characters.villians}{characters.antiHeroes}</ListBox>
+
+### Adding a preselected option to the ListBox
+
+You can have the Listbox display a preselected option by using the ```defaultIsSelected``` property on the ```ListBox.option```. Set it equal to the option you wish to display.
+
+```js
+  <ListBox
+    isInline
+    onChange={activeOptionIndex => {
+      console.log(activeOptionIndex);
+    }}
+  >
+    {characters.antiHeroesRaw.map((item, index) => {
+      return (
+        <ListBox.Option key={item.label} defaultIsSelected={index === 4}>
+          {item.label}
+        </ListBox.Option>
+      );
+    })}
+  </ListBox>
+```
+
+#### Result
+
+<ListBox isInline onChange={activeOptionIndex => {console.log(activeOptionIndex);}}>{characters.antiHeroesRaw.map((item, index) => {return (<ListBox.Option key={item.label} defaultIsSelected={index === 4}>{item.label}</ListBox.Option>);})}</ListBox>
+
+### Multi ListBox
+
+The `<ListBox>` component provides a prop `isMulti`. It allows users to select multiple items from a drop-down list.
```

Comment:
perhaps stay away from "drop-down" So it doesn't get confused with our DropdownMenu component.